README.md

```diff
@@ -32,7 +29,7 @@ Read [The Orderbook Manifesto](https://manifest.trade/whitepaper.pdf)
 | Feeless |No |No |Yes|
 | Atomic lot sizes |No |No |Yes|
 | Anchor |Yes |No|No|
-| Creation Rent|2 SOL |3+ SOL |.004 SOL|
+| Creation Rent|2 SOL |3+ SOL |.007 SOL|
```

Comment:
This is because you need to create 2 token accounts? Seems external to the market imo